src/resources/filters/modules/typst_css.lua

```diff
@@ -593,6 +593,10 @@ local function quote(s)
   return '"' .. s .. '"'
 end
 
+local function dequote(s)
+  return s:gsub('^["\']', ''):gsub('["\']$', '')
+end
```

Comment:
I think this is safe here, but it's always terrifying to consider the full implications of quoting, escaping, etc. There's no chance that fonts can have `,` in their titles, right?

Reply:
(this is also about L662 below)

Reply:
:shudder:

The spec does not seem to ban commas in font family names. In practice this seems to be something that happens in PostScript because of weights and styles of a family, but CSS handles this differently (and better).

Reply:
I conclude that we shouldn't get font names with commas in them from CSS. But if we run into trouble with this later, I'd be glad to write a PEG parse for parsing font lists.